sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -114,7 +114,6 @@ async def serialize_property(value: 'Input[Any]',
 
     if known_types.is_resource(value):
         resource = cast('Resource', value)
-        deps.append(resource)
```

Comment:
What's the context for this change?

Reply:
This `append` is not necessary, as this branch of the marshaller bottoms out in marshaling other properties which will add the resource to the dependency list.